otc-watcher/pkg/api/api.go

```diff
@@ -4,51 +4,52 @@ import (
 	"encoding/json"
 	"net/http"
 
-	"github.com/skycoin/services/otc-watcher/pkg/scanner"
+	"log"
+
 	"github.com/skycoin/services/otc/pkg/otc"
+
+	"github.com/skycoin/services/otc-watcher/pkg/scanner"
 )
 
 func New(scnr *scanner.Scanner) *http.ServeMux {
 	mux := http.NewServeMux()
-	mux.HandleFunc("/register", Register(scnr))
 	mux.HandleFunc("/outputs", Outputs(scnr))
 	return mux
 }
 
-func Register(scnr *scanner.Scanner) http.HandlerFunc {
-	return func(w http.ResponseWriter, r *http.Request) {
-		var (
-			req *otc.Drop
-			err error
-		)
-
-		if err = json.NewDecoder(r.Body).Decode(&req); err != nil {
-			http.Error(w, "invalid JSON", http.StatusBadRequest)
-			return
-		}
-
-		if err = scnr.Register(req); err != nil {
-			http.Error(w, "server error", http.StatusInternalServerError)
-			return
-		}
-	}
-}
-
 func Outputs(scnr *scanner.Scanner) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		var (
 			outputs otc.Outputs
 			req     *otc.Drop
-			err     error
+			err1    error
```

Comment:
there's no reason to create `err1` and `err2` -- just use `err` and overwrite the previous value (how it was before)